### PR TITLE
test: added test for tvg_picture_get_viewbox function

### PR DESCRIPTION
**Pull request by bchrescionko**
Added lines for testing tvg_picture_get_viewbox function in test/capi/capiPicture.cpp

### CODE REVIEW

**Comment by mgrudzinska** [comment]
@hermet, since the viewbox api is in beta, I assume the corresponding capi also will be marked as such. so do we want to include it into tests?

**Comment by hermet** [comment]
@mgrudzinska there is no point to keep the deprecated apis test, but for the beta apis, it's not recommended neither. But I think we can apply it because we have no concrete plans to promoting/dropping it.

File: test/capi/capiPicture.cpp [hunk 1]
```diff
@@ -66,6 +66,12 @@ TEST_CASE("Load Svg Data in Picture", "[capiPicture]")
     REQUIRE(w == Approx(1000).epsilon(0.0000001));
     REQUIRE(h == Approx(1000).epsilon(0.0000001));
 
+    //Verify Position
+    float x, y;
+    REQUIRE(tvg_picture_get_viewbox(picture, &x, &y, &w, &h) == TVG_RESULT_SUCCESS);
+    REQUIRE(x == Approx(0).epsilon(0.0000001));
+    REQUIRE(y == Approx(0).epsilon(0.0000001));
```
Comment (hermet) [inline test/capi/capiPicture.cpp:73]: btw, why do you only check x/y, not w/h?

**Review by hermet — COMMENTED** [review]
please check a comment.

File: test/capi/capiPicture.cpp (hunk 1 above)
Comment (bchrescionko) [reply test/capi/capiPicture.cpp:73]: Becouse it was check with with tvg_picture_get_size in line 65, so I didn't if I had to check it again

Comment (hermet) [reply test/capi/capiPicture.cpp:73]: Unit test is to verify the apis work correctly or not.
In point of API perspective view, tvg_picture_get_size() and tvg_picture_get_viewbox() have no relationship with each others.
That means it needs test possible scenarios for tvg_picture_get_viewbox() return values.

Thus, please check all return values.

**Review by hermet — CHANGES_REQUESTED** [review]
Please check my comments.

File: test/capi/capiPicture.cpp (hunk 1 above)
Comment (bchrescionko) [reply test/capi/capiPicture.cpp:73]: Ok, I wll check it

**Review by hermet — APPROVED** [review]

**Pull request merged**